Remove duplicate assignment

nmtigor opened the pull request:
There is a same assignment above in the App.constructor.

nmtigor committed:
Fix orders of property chains of Operators in AstBinaryOperator's methods.

calixteman committed:
[Editor] Make sure to have the annotation editor layer on top of the annotation one (bug 1791515)

Some z-index have been added in the annotation layer because the elements inside are re-ordered
in order to improve accessibility.
Hence we must add a "high" z-index on the annotation editor layer in order to avoid any bad
interaction between the different layers.

nmtigor committed:
Remove duplicate assignment in App.constructor.

Snuffleupagus commented:
Please provide a slightly more detailed commit message.

nmtigor committed:
Fix commFunc typo in the SandboxSupportBase.destroy method

nmtigor committed:
Fix #selectEditors typo in the AnnotationEditorUIManager.unselectAll method

Snuffleupagus commented:
Please provide a slightly more detailed *commit message* (in addition to the PR description); see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

calixteman committed:
[XFA] Fix an hidden issue in the FormCalc lexer

Since there are no script engine with XFA, the FormCalc parser is not used irl.
The bug @nmtigor noticed was hidden by another one (the wrong check on `match`).

Snuffleupagus committed:
Access the `setTimeout`-functionRefs correctly in `SandboxSupportBase.destroy`

*This effectively replaces PR 15465.*

As outlined in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/forEach, the argument order when iterating through a `Map` is actually `value, key`.
Ignoring the incorrect Array used in the old code, I cannot imagine that this would've worked anyway since we didn't use the actual `setTimeout`-functionRefs to clear the timeouts; please refer to the `setTimeout`/`setInterval` methods in the `SandboxSupportBase.createSandboxExternals` method.

calixteman committed:
Initialize values in the path bounding box before flushing the operator list (bug 1791583)

OperatorList.addOp can trigger a flush if it's required, hence the values passed to it must
be correctly initialized in order to avoid some wrong values in the renderer.
Because of that a clip path was considered as empty, nothing was clipped, hence the wrong
rendering in bug 1791583.

Snuffleupagus commented:
I've resubmitted this in PR #15483, with the commit history (and message) cleaned-up but *obviously* keeping the original author information intact.
